src/attack.c

```diff
@@ -1065,17 +1081,18 @@ attack_network (struct scan_globals *globals)
     connect_main_kb (&alive_hosts_kb);
 
   gettimeofday (&then, NULL);
-
   if (check_kb_access ())
-    return;
-
+    {
+      g_error ("No access to redis kb");
```

Comment:
Don't use g_error, as suggested in [here](https://developer-old.gnome.org/glib/stable/glib-Message-Logging.html#g-error)
Use g_warning instead, or handle the error with GError.